### ISSUE TITLE
Wrong syntax used for function examples with a params object on docs.enonic.com

### ISSUE THREAD

**Issue by cfenzo** [enonic/xp#5647]
Hi,
As the title says, wrong syntax is used for function examples with a params object on docs.enonic.com.
It gives the impression that the function is to be called with multiple (and some optional) arguments, in stead of just one argument: an object with (some optional) properties.

[Wrong syntax, used on docs.enonic.com](http://docs.enonic.com/en/stable/developer/assets.html):
```js
assetUrl(path [,application] [,type] [,params])
```
[Correct syntac, used in js docs](http://repo.enonic.com/public/com/enonic/xp/docs/6.11.1/docs-6.11.1-libdoc.zip!/module-portal.html#.assetUrl):
```js
assetUrl({
    path: 'styles/main.css'
});
```

Same goes for [serviceUrl](http://docs.enonic.com/en/stable/developer/services/index.html#access) and probably any other place with examples of functions using a params object.

**Comment by jsi** [comment]
I believe this has been fixed now.

**Comment by jsi** [comment]
It was fixed for the 6.10, 6.11 and latest versions of the doc.
